{{cookiecutter.project_name}}/src/manage.py

```diff
@@ -26,7 +26,7 @@ def main():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pipit.settings.prod")
 
     if_exists_load_env(".env")
-    if_exists_load_env("../local.env")
+    if_exists_load_env(".env.local")
```

Comment:
Maybe it's time/better to retire src/.env and only go for src/.env.local?

Reply:
I was actually going to write that `if_exists_load_env(".env")` is used in production, but that work is actually done by uwsgi. So technically, for us at Fröjd, its never used.

With that said, I think we should keep `.env` because I think users expect the functionality to be there.